Makefile

```diff
@@ -77,7 +78,21 @@ appimage-amd64: build/universal-robots
 
 appimages: appimage-amd64 appimage-arm64
 
-.PHONY: module.tar.gz
 module.tar.gz: meta.json
 	cp ./packaging/appimages/deploy/universal-robots-latest-$(ARCH).AppImage universal-robots.AppImage
 	tar czf $@ $^ universal-robots.AppImage
+
+build/_deps/universal_robots_client_library-src/scripts/start_ursim.sh: build
+	# we need to ignore `cmake -G Ninja  ..` failing as the this project's CMakeLists.txt 
+	# assumes that the viam sdk is available as a system package (which won't be true on most
+	# development machines).
+	# However even though it fails, it will still download theuniversal-robots SDK repo which
+	# is all we need to run the sim.
+	cd build && \
+	cmake -G Ninja  ..  || true
```

Comment:
does the` || true` make this target always return success? def okay with that just want to know what behavior, since the comment implies this target will always fail

Reply:
The target always succeeds despite the cmake command always failing

Reply:
So yes